### DISCUSSION TITLE
How to apply the function of automatically generating swagger documentation in tsed to other projects?

### DISCUSSION THREAD

**Discussion by FuHongBiao** [TypedProject/ts-express-decorators#1940] [Q&A]
hello every one.I have a thinkjs project now, I want to use tsed's auto-generate swagger documentation function, how can I do it?

**Comment by Romakita** [comment]
You have to use Ts.ED decorators from `@tsed/schemaˋ on a class. I don’t use thinkjs so I don’t know how this framework works.

getSpec(MyController) return a partial openspec schema.

See you
